cobalt/dom/performance_resource_timing.idl

```diff
@@ -29,4 +29,5 @@ interface PerformanceResourceTiming : PerformanceEntry {
   readonly  attribute unsigned long long  transferSize;
   readonly  attribute unsigned long long  encodedBodySize;
   readonly  attribute DOMHighResTimeStamp workerStart;
+  readonly  attribute DOMString           nextHopProtocol;
```

Comment:
Tiny nit but in the spec / chrome version of this file it's after initiatiorType and before fetchStart. Maybe nice to keep it consistent ?